app/controllers/permissions_controller.rb

```diff
@@ -104,84 +72,53 @@ def create
       permission_creation_error = PermissionsHelper::ErrorCodeMessages[response.status]
       permission_creation_error ||= Array.wrap(response.body['errors'])[0]
 
-      flash.now[:error] = permission_creation_error.capitalize
```

Comment:
We've long `battled` about modifying error messages (minor or major), if we need error messages updated we should file tickets with the appropriate project.